### PR TITLE
Make approvals_before_merge to be nullable

### LINKED ISSUE #226: JsonDecodingException on approvals_before_merge
Hello. I'm catching an exception, because `approvals_before_merge` field is `null`. According to GitLab doc this field is not required in the response and can be `null`.
```
Compiling Dangerfile Dangerfile.df.kts
Executing kotlinc -script-templates systems.danger.kts.DangerFileScript -cp /usr/local/lib/danger/danger-kotlin.jar -script Dangerfile.df.kts /var/folders/5q/4nc5x9yd6mvcpzd_g_cwjdyc0000gn/T/danger-dsl.json danger_out.json - pid 78358

kotlinx.serialization.json.internal.JsonDecodingException: Unexpected JSON token at offset 95589: Unexpected symbol 'n' in numeric literal
JSON input: .....    "approvals_before_merge": null,
        "subscribed": fa.....
```

### PR DESCRIPTION
**Pull request by pinguinjkeke**
Details described in #226

### CODE REVIEW

**Review by f-meloni — APPROVED** [review]
Thank you :)

File: danger-kotlin-library/src/main/kotlin/systems/danger/kotlin/models/gitlab/GitLab.kt [hunk 1]
```diff
@@ -38,7 +38,7 @@ data class GitLabMergeRequest(
     @SerialName("allow_maintainer_to_push")
     val allowMaintainerToPush: Boolean = false,
     @SerialName("approvals_before_merge")
-    val approvalsBeforeMerge: Int = 0,
+    val approvalsBeforeMerge: Int? = 0,
```
Comment (f-meloni) [inline danger-kotlin-library/src/main/kotlin/systems/danger/kotlin/models/gitlab/GitLab.kt:41]: Can you please also update the CHANGELOG?

Comment (pinguinjkeke) [reply danger-kotlin-library/src/main/kotlin/systems/danger/kotlin/models/gitlab/GitLab.kt:41]: Done

Comment (pinguinjkeke) [reply danger-kotlin-library/src/main/kotlin/systems/danger/kotlin/models/gitlab/GitLab.kt:41]: It fails on `e: Unable to compile C bridges`

**Pull request merged**